cmd/status/cmdstatus.go

```diff
@@ -197,26 +199,31 @@ func (r *Runner) listInvFromCluster() (*printer.PrintData, error) {
 	// initialize maps in printData
 	printData := printer.PrintData{
 		Identifiers: object.ObjMetadataSet{},
-		InvNameMap:  make(map[object.ObjMetadata]string),
+		InvNameMap:  make(map[object.ObjMetadata]fmt.Stringer),
 		StatusSet:   r.statusSet,
 	}
 
-	identifiersMap, err := invClient.ListClusterInventoryObjs(r.ctx)
+	inventories, err := invClient.List(r.ctx, inventory.ListOptions{})
 	if err != nil {
 		return nil, err
 	}
 
-	for invName, identifiers := range identifiersMap {
+	identifiersMap := make(map[inventory.ID]object.ObjMetadataSet)
+	for _, inv := range inventories {
+		identifiersMap[inv.ID()] = inv.ObjectRefs()
+	}
+
+	for invID, identifiers := range identifiersMap {
 		// Check if there are targeted inventory names and include the current inventory name
-		if _, ok := r.inventoryNameSet[invName]; !ok && len(r.inventoryNameSet) != 0 {
+		if _, ok := r.inventoryNameSet[invID.String()]; !ok && len(r.inventoryNameSet) != 0 {
 			continue
 		}
 		// Filter objects
 		for _, obj := range identifiers {
 			// check if the object is under one of the targeted namespaces
 			if _, ok := r.namespaceSet[obj.Namespace]; ok || len(r.namespaceSet) == 0 {
 				// add to the map for future reference
-				printData.InvNameMap[obj] = invName
+				printData.InvNameMap[obj] = invID
```

Comment:
InvIDMap?

Reply:
I guess the question is: Does the printer print them as IDs or Names?

If it prints names, how and where do we map ID to name?

If it prints IDs, then it should say ID throughout.

Are we breaking the printer by using the ID instead of the Name?

Reply:
As you mentioned in chat, the kapply CLI only uses the single-object inventory, so it always prints the name. So it's ok if the fields in the CLI command all say Name. 

Perhaps this should cast the Info to the SingleObjectInfo and use the Name method then, instead of needing to explain how it gets the name from the ID?